syrillian/src/engine/components/skybox.rs

```diff
@@ -0,0 +1,231 @@
+use crate::World;
```

Comment:
Isn't this missing the whole RenderProxy integration? Did you test if you can use a cubemap as a Skybox? I don't see how the Skybox Component is able to do anything at all.

Reply:
You're correct. Will add the RenderProxy integration and test it properly. Thanks for pointing this out.